pkg/infra/game/agent/random.go

```diff
@@ -0,0 +1,23 @@
+package agent
+
+import (
+	"infra/game/decision"
+	"infra/game/state"
+	"math/rand"
+)
+
+type RandomAgent struct {
+}
+
+func (r RandomAgent) HandleFight(state state.State, baseAgent BaseAgent) {
+	fight := rand.Intn(1)%2 == 0
```

Comment:
Should probably just be `fight := rand.Intn(2) == 0` right?

Reply:
Yes I think we did change it to that but it must've slipped through the commit.